src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -779,7 +779,7 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
                 // when opponent accept user's take back
                 await receivePartDAOUpdate(BASE_TAKE_BACK_REQUEST);
                 const opponentTurnDiv: DebugElement = componentTestUtils.findElement('#currentPlayerIndicator');
-                expect(opponentTurnDiv.nativeElement.innerText).toBe(`It is your turn.`);
+                expect(opponentTurnDiv.nativeElement.innerText).toBe(`It is creator's turn.`);
```

Comment:
ah bon ?

Reply:
c'était toi qui avait changé le display de "your" en {{username}} non?

Reply:
Bah, je vois ici l'avant après qui semble indiquer qu'avant c'était "it is your turn" et qu'après tes changements c'était ...{{ username }}....
Non ?

Reply:
vrai vrai sorry! Cependant, ici c'est bien un fix du test non ? Ce test fait:
  1. Recevoir un mouvement
  2. Envoyer un mouvement
  3. Recevoir une demande de take back
  4. Accepter
On retourne donc au tour du premier joueur (avant 1.), qui n'est pas nous vu qu'on a *reçu* le premier mouvement ! Et le jeu est démarré par OPPONENT (comme dans le test de base), donc c'est le tour de creator : "it is creator's turn", et pas "it is your turn" 

Qu'il marche en develop et pas dans cette branche c'est suspect, mais selon le raisonnement précédent ça me semble correct ainsi ?

Reply:
euuuh oui j'me suis bien embrouillé ici, dans ma correction aussi mais... C'est de retour à la normale et les tests passent !